index.js

```diff
@@ -146,6 +147,22 @@ function logger(options) {
     options.colorStatus = options.colorStatus || false;
     options.expressFormat = options.expressFormat || false;
     options.ignoreRoute = options.ignoreRoute || function () { return false; };
+    options.nginxFormat = options.nginxFormat || false;
```

Comment:
This should probably be `clfFormat` because it's used by (at least) both Apache and nginx.